draft/features-roadmap.rst

```diff
@@ -0,0 +1,116 @@
+=========================
+CEP XXXX: Feature Roadmap
+=========================
+
+:CEP: XXXX
+:Author: Omer Katz
+:Implementation Team: Omer Katz
+:Shepherd: Omer Katz
```

Comment:
this could be celery core team+technical board

Reply:
I assigned myself because at the time we didn't have anyone else active on this repository.

We'll decide who shepherds this soon.

Reply:
fair enough